.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
- [ Optional] Please fix the weried margin at the right of the project in the mobile version the mobile versio shouldn't affect by the desktop version.
![image](https://github.com/melkamu12/Portfolio-DesktopVersionM/assets/74744787/b2cbbd16-2df3-4802-bf6e-40a944e4c5c3)